api-examples/pretrain-transformer-lm.py

```diff
@@ -579,11 +589,28 @@ def train():
         for i, batch in enumerate(train_loader):
             x, y = batch
             inputs = {'x': x.to(args.device)}
-            labels = y.to(args.device).transpose(0, 1).contiguous()
+            labels = y.to(args.device)
+            if args.mlm:
+                # Replace 15% of tokens
+                masked_indices = torch.bernoulli(torch.full(labels.shape, 0.15)).byte()
```

Comment:
In BERT are all the batches totally packed and there fore we don't have to worry about the thing getting masked is in the padding region?

Reply:
Yeah, this shouldnt be a concern